tools/upgrade_versions.py

```diff
@@ -28,6 +28,7 @@
 from lib import rpm_package
 from lib.utils import replace_str_in_file
 from lib.versions_repository import setup_versions_repository
+from tools.update_versions_readme import update_versions_in_readme
```

Comment:
I was just trying to alter Lucas' patch as less as possible. Done.

TBH I do not understand why we have this tools directory, IMHO entire code would be in lib and executable scripts would simply call them, as we had in PowerKVM project.